docs/reference/java-interop.md

```diff
@@ -333,15 +356,14 @@ if (Character.isLetter(a)) {
 ### Java Reflection
 
 Java reflection works on Kotlin classes and vice versa. As mentioned above, you can use `instance.javaClass` or 
-`javaClass<ClassName>()` to enter Java reflection through `java.lang.Class`. You can then "convert" to Kotlin reflection
+`ClassName::class.java` to enter Java reflection through `java.lang.Class`. You can then "convert" to Kotlin reflection
 by calling `.kotlin`:
```

Comment:
This suggestion no longer makes sense :) To get to Kotlin reflection I'd just use `ClassName::class`, no need to do `.java` and then `.kotlin`